### DISCUSSION TITLE
AppArmor profile suggested after upgrade, but how to?

### DISCUSSION THREAD

**Discussion by tigexplorer** [filips123/PWAsForFirefox#577] [Q&A] [answered]
After upgrading my machine to ubuntu 24, the firefox was changed from the ppa to snap.

My PWA's works fine, despite this change.

But now, I get this message in every PWA:

`Some of Firefox’s security features may offer less protection on your current operating system`

with a link to:

https://support.mozilla.org/en-US/kb/install-firefox-linux?as=u&utm_source=inproduct

But I'm unsure, if the solution provided on this page is really the right choice for the PWA's and don't interfere with other AppArmor profiles, already created automatically for firefox?

**Comment by filips123** [comment, marked as answer]
You can follow the instructions from that link for creating AppArmor profile. You can name it something like `firefox-pwa`, and the path to Firefox should be `/home/<USER>/.local/share/firefoxpwa/runtime/{firefox,firefox-bin,updater}` (or something similar, depending on your [user data](https://pwasforfirefox.filips.si/resources/installation-directories/#user-data) directory).

**Comment by tigexplorer** [comment]
Yeah, it seems to work!
Thanx for your quick reply an your commitment!
